xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -218,11 +218,11 @@
   
  Any callbacks or cancelable operations registered with the <xref:System.Threading.CancellationToken> will be executed.  
   
- Cancelable operations and callbacks registered with the token should not throw exceptions.  
+ It is recommended that cancelable operations and callbacks registered with <xref:System.Threading.CancellationToken> do not throw exceptions. However, this overload of Cancel will aggregate any exceptions thrown into an <xref:System.AggregateException>, such that one callback throwing an exception will not prevent other registered callbacks from being executed. 
+
+ The <xref:System.Threading.ExecutionContext> that was captured when each callback was registered will be reestablished when the callback is invoked.   
   
- However, this overload of Cancel will aggregate any exceptions thrown into an <xref:System.AggregateException>, such that one callback throwing an exception will not prevent other registered callbacks from being executed.  
-  
- The <xref:System.Threading.ExecutionContext> that was captured when each callback was registered will be reestablished when the callback is invoked.  
+ If <xref:System.Threading.SynchronizationContext> was captured when a callback was registered, that callback will be executed on the captured context.  
```

Comment:
I would say "via" rather than "on", but otherwise it's ok.

I'm not sure this extra SynchronizationContext discussion adds much value (and to my knowledge almost no one uses the support); seems like the current docs that describe the parameter would be sufficient.  But I don't have a strong opinion either way.

Reply:
@stephentoub, would it be appropriate to drop the mentions of _both_ SynchronizationContext and ExecutionContext here (but leave them on [CancellationToken.Register](https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.register?view=netcore-2.0))?

My goal with this edit was to bring symmetry. [CancellationToken.Register](https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.register?view=netcore-2.0) mentions both contexts; here, only one is mentioned. We could bypass this inconsistency by not mentioning either context here.

What do you think...good idea or bad idea? :-)

Reply:
I'd be fine with that.

Reply:
Thanks! Change made.